### ISSUE TITLE
Spyder does not start after the first time as the spyder-terminal library does not work on my system

### ISSUE THREAD

**Issue by b-a0** [spyder-ide/spyder#21360]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
A fresh install of Spyder 5.4.5 on Windows Server 2016 version 1607 starts the first time with the following error:

![image](https://github.com/spyder-ide/spyder/assets/25707742/2a2d798e-e240-49a6-bf9c-4c0e209108be)

> It was not possible to load the Terminal plugin. The problem was:
>
> Unfortunately, the library that spyder-terminal uses to create terminals is failing to
work in your system. Therefore, this plugin will be deactivated.
>
> This usually happens on Windows 7 systems. If that's the case, please consider
updating to a newer Windows version.

All fine, I can work without the terminal. Spyder starts up, without a terminal panel and everything seems to work.
However, when I restart Spyder the same error appears, but this time Spyder does not launch and the error disappears quickyl by itself. I can no longer launch Spyder.

### What steps reproduce the problem?

1. Launch freshly installed Spyder (without a `~/.spyder-py3` directory from older versions)
2. Click OK on the Plguin compatibility check error
3. Close Spyder
4. Launch Spyder

### What is the expected output? What do you see instead?

I would expect that Spyder starts the second time, without showing the error and without the terminal panel.  
Instead, Spyder starts launching, shows the splash screen, shows the error and then crashes. 

I cannot launch Spyder, unless I delete the `~/.spyder-py3` directory such that it appears to be a "fresh" install. Then the cycle repeats.

### Paste Traceback/Error Below (if applicable)

When launching using the commandline: 

```powershell
PS C:\Program Files\Spyder> .\Python\python.exe .\Spyder.launch.pyw --debug-info verbose
```

I get the following log (I only included that last few lines before the crash:

<details>
<summary>Fold out launch log </summary>

```log
2023-09-21 14:23:02,918 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> nontext_ratio1: 0.9912109375
2023-09-21 14:23:02,918 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> nontext_ratio2: 0.9833984375
2023-09-21 14:23:02,919 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> is_likely_binary: True
2023-09-21 14:23:02,920 [DEBUG] [chardet.charsetprober] -> SHIFT_JIS Japanese prober hit error at byte 105
2023-09-21 14:23:02,921 [DEBUG] [chardet.charsetprober] -> EUC-JP Japanese prober hit error at byte 105
2023-09-21 14:23:02,921 [DEBUG] [chardet.charsetprober] -> GB2312 Chinese prober hit error at byte 105
2023-09-21 14:23:02,921 [DEBUG] [chardet.charsetprober] -> EUC-KR Korean prober hit error at byte 105
2023-09-21 14:23:02,922 [DEBUG] [chardet.charsetprober] -> CP949 Korean prober hit error at byte 105
2023-09-21 14:23:02,922 [DEBUG] [chardet.charsetprober] -> Big5 Chinese prober hit error at byte 105
2023-09-21 14:23:02,922 [DEBUG] [chardet.charsetprober] -> EUC-TW Taiwan prober hit error at byte 105
2023-09-21 14:23:02,923 [DEBUG] [chardet.charsetprober] -> Johab Korean prober hit error at byte 105
2023-09-21 14:23:02,925 [DEBUG] [chardet.charsetprober] -> windows-1251 Russian confidence = 0.12799495392693638
2023-09-21 14:23:02,925 [DEBUG] [chardet.charsetprober] -> KOI8-R Russian confidence = 0.01
2023-09-21 14:23:02,925 [DEBUG] [chardet.charsetprober] -> ISO-8859-5 Russian confidence = 0.01
2023-09-21 14:23:02,925 [DEBUG] [chardet.charsetprober] -> MacCyrillic Russian confidence = 0.043998265412384385
2023-09-21 14:23:02,926 [DEBUG] [chardet.charsetprober] -> IBM866 Russian confidence = 0.0
2023-09-21 14:23:02,932 [DEBUG] [chardet.charsetprober] -> IBM855 Russian confidence = 0.01
2023-09-21 14:23:02,932 [DEBUG] [chardet.charsetprober] -> ISO-8859-7 Greek confidence = 0.0
2023-09-21 14:23:02,932 [DEBUG] [chardet.charsetprober] -> windows-1253 Greek confidence = 0.0
2023-09-21 14:23:02,932 [DEBUG] [chardet.charsetprober] -> ISO-8859-5 Bulgarian confidence = 0.0
2023-09-21 14:23:02,932 [DEBUG] [chardet.charsetprober] -> windows-1251 Bulgarian confidence = 0.0
2023-09-21 14:23:02,933 [DEBUG] [chardet.charsetprober] -> TIS-620 Thai confidence = 0.01
2023-09-21 14:23:02,933 [DEBUG] [chardet.charsetprober] -> ISO-8859-9 Turkish confidence = 0.0
2023-09-21 14:23:02,933 [DEBUG] [chardet.charsetprober] -> windows-1255 Hebrew confidence = 0.0
2023-09-21 14:23:02,933 [DEBUG] [chardet.charsetprober] -> windows-1255 Hebrew confidence = 0.0
2023-09-21 14:23:02,933 [DEBUG] [chardet.charsetprober] -> windows-1255 Hebrew confidence = 0.0
2023-09-21 14:23:02,933 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> detected_encoding: {'encoding': 'MacRoman', 'confidence': 0.5153771760154738, 'language': ''}
2023-09-21 14:23:02,934 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> failure: decodable_as_unicode: False
2023-09-21 14:23:02,940 [DEBUG] [spyder.utils.external.binaryornot.check] -> is_binary: 'C:\\Users\\username\\ntuser.pol'
2023-09-21 14:23:02,944 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> nontext_ratio1: 0.5302734375
2023-09-21 14:23:02,945 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> nontext_ratio2: 1.0
2023-09-21 14:23:02,945 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> is_likely_binary: False
2023-09-21 14:23:02,946 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> detected_encoding: {'encoding': 'utf-16le', 'confidence': 0.85, 'language': ''}
2023-09-21 14:23:02,946 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> failure: decodable_as_unicode: False
2023-09-21 14:23:02,946 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> has nulls:True
2023-09-21 14:23:03,139 [DEBUG] [spyder.utils.external.binaryornot.check] -> is_binary: 'C:\\Users\\username\\pspp.jnl'
2023-09-21 14:23:03,140 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> nontext_ratio1: 0.0
2023-09-21 14:23:03,143 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> nontext_ratio2: 1.0
2023-09-21 14:23:03,143 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> is_likely_binary: False
2023-09-21 14:23:03,143 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> detected_encoding: {'encoding': 'ascii', 'confidence': 1.0, 'language': ''}
2023-09-21 14:23:03,144 [DEBUG] [spyder.utils.external.binaryornot.helpers] -> failure: decodable_as_unicode: False
2023-09-21 14:23:03,152 [DEBUG] [spyder.utils.external.binaryornot.check] -> is_binary: 'C:\\Users\\username\\spyder_terminal_err.log'
2023-09-21 14:23:03,174 [DEBUG] [urllib3.connectionpool] -> Starting new HTTP connection (1): 127.0.0.1:8084
2023-09-21 14:23:04,178 [INFO] [spyder.app.mainwindow] -> Loading terminal...
2023-09-21 14:23:04,221 [DEBUG] [spyder.api.plugin_registration.registry] -> Plugin preferences has already loaded
2023-09-21 14:23:04,222 [DEBUG] [spyder.api.plugin_registration.mixins] -> Calling <bound method TerminalPlugin.on_preferences_available of <spyder_terminal.terminalplugin.TerminalPlugin object at 0x00000249D158D5E0>>
2023-09-21 14:23:04,225 [DEBUG] [spyder.api.plugin_registration.registry] -> Plugin editor has already loaded
2023-09-21 14:23:04,225 [DEBUG] [spyder.api.plugin_registration.mixins] -> Calling <bound method TerminalPlugin.on_editor_available of <spyder_terminal.terminalplugin.TerminalPlugin object at 0x00000249D158D5E0>>
2023-09-21 14:23:04,226 [DEBUG] [spyder.api.plugin_registration.registry] -> Plugin project_explorer has already loaded
2023-09-21 14:23:04,226 [DEBUG] [spyder.api.plugin_registration.mixins] -> Calling <bound method TerminalPlugin.on_projects_available of <spyder_terminal.terminalplugin.TerminalPlugin object at 0x00000249D158D5E0>>
2023-09-21 14:23:04,226 [DEBUG] [spyder.api.plugin_registration.registry] -> Plugin workingdir has already loaded
2023-09-21 14:23:04,226 [DEBUG] [spyder.api.plugin_registration.mixins] -> Calling <bound method TerminalPlugin.on_working_directory_available of <spyder_terminal.terminalplugin.TerminalPlugin object at 0x00000249D158D5E0>>
2023-09-21 14:23:04,226 [INFO] [spyder.app.mainwindow] -> Loading old third-party plugins...
2023-09-21 14:23:04,253 [INFO] [spyder.app.mainwindow] -> Creating Menus...
2023-09-21 14:23:04,265 [INFO] [spyder.app.mainwindow] -> Creating toolbars...
2023-09-21 14:23:04,265 [INFO] [spyder.app.mainwindow] -> Setting up main window...
2023-09-21 14:23:04,271 [INFO] [spyder.app.mainwindow] -> Setting up window...
2023-09-21 14:23:04,272 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Spyder Default Layout') (Spyder Default Layout) under context __global for plugin layout
2023-09-21 14:23:04,272 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Rstudio layout') (Rstudio layout) under context __global for plugin layout
2023-09-21 14:23:04,273 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Matlab layout') (Matlab layout) under context __global for plugin layout
2023-09-21 14:23:04,273 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Horizontal split') (Horizontal split) under context __global for plugin layout
2023-09-21 14:23:04,273 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Vertical split') (Vertical split) under context __global for plugin layout
2023-09-21 14:23:05,165 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Spyder Default Layout') (Spyder Default Layout) under context __global for plugin layout
2023-09-21 14:23:05,169 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Rstudio layout') (Rstudio layout) under context __global for plugin layout
2023-09-21 14:23:05,172 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Matlab layout') (Matlab layout) under context __global for plugin layout
2023-09-21 14:23:05,176 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Horizontal split') (Horizontal split) under context __global for plugin layout
2023-09-21 14:23:05,179 [DEBUG] [spyder.utils.registries] -> Registering SpyderAction('Vertical split') (Vertical split) under context __global for plugin layout
2023-09-21 14:23:05,183 [DEBUG] [spyder.plugins.layout.plugin] -> Tabifying new plugins
2023-09-21 14:23:05,184 [INFO] [spyder.plugins.layout.plugin] -> Tabifying terminal plugin for the first time next to [<spyder.plugins.ipythonconsole.plugin.IPythonConsole object at 0x00000249CCC0BCA0>]
```

</details>

If I look in `spyder_terminal_err.log` which is referred in the log, I see this (the weird characters are there in the actual logfile):

```log
No QCoreApplication instance found. Application patches not applied. You have to call load_stylesheet function after instantiation of QApplication to take effect. 
[32m2023-09-21 14:23:05[0m [35mAPP-REKEN05[0m [34m__main__[32380][0m [1;30mINFO[0m Server is now at: 127.0.0.1:8084
[32m2023-09-21 14:23:05[0m [35mAPP-REKEN05[0m [34m__main__[32380][0m [1;30mINFO[0m Shell: C:\Windows\system32\cmd.exe
[32m2023-09-21 14:25:00[0m [35mAPP-REKEN05[0m [34m__main__[32380][0m [1;30mINFO[0m Closing server...

C:\Program Files\Spyder\pkgs\spyder_terminal\server\__main__.py:53: RuntimeWarning: coroutine 'TermManagerBase.shutdown' was never awaited
  application.term_manager.shutdown()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```

## Versions

* Spyder version: 5.4.5
* Python version: The one shipped with Spyder
* Qt version: The one shipped with Spyder
* PyQt version: The one shipped with Spyder
* Operating System name/version: Windows Server 2016 version 1607

**Comment by b-a0** [comment]
Forgot to mention: a workaround I found is that I can edit `~\.spyder-py3\config\spyder.ini` and add

```txt
[terminal]
enable = False
```

However, this is not feasible as many users use this Spyder install and I would then have to set this key for all of them (or instruct them to do it themselves).

**Comment by ccordoba12** [comment]
Hey @b-a0, thanks for reporting. We're aware of this problem and that's why we decided to stop packaging Spyder-terminal along with our Windows and Mac installers some weeks ago (see pull request #21266).

That change will be available in our next version (5.5.0), to be released next month. In the meantime, please keep using the workaround you found to avoid it.
